test/e2e_new/config/aws-dev.yaml

```diff
@@ -0,0 +1,104 @@
+---
+# E2E test scenario using local dev images and manifests built from the source tree for following providers:
+# - cluster-api
+# - bootstrap kubeadm
+# - control-plane kubeadm
+# - aws
+
+# For creating local dev images built from the source tree;
+# - from the CAPI repository root, `make docker-build REGISTRY=gcr.io/k8s-staging-cluster-api` to build the cluster-api, bootstrap kubeadm, control-plane kubeadm provider images.
+# - from the CAPA repository root, `make docker-build REGISTRY=gcr.io/k8s-staging-cluster-api` to build the aws provider image.
+
+# Following env variables should be set before running the provider
+
+# export AWS_REGION=
+# export AWS_SSH_KEY_NAME=
+# export AWS_B64ENCODED_CREDENTIALS=
```

Comment:
Can we create AWS_B64ENCODED_CREDENTIALS internally using `clusterawsadm` and just rely on the aws env vars for the test instead? That will translate much easier to the prow environment and remove a manual step for developers.

Reply:
will need to be refactored on clusterawsadm v2, which i have done a fair amount for #1734 . I could fix this up post-merge?